build-core-armv7a7-imx6ull.sh

```diff
@@ -45,10 +45,8 @@ fi
 b_log "Building posixsrv"
 make -C "phoenix-rtos-posixsrv" all install
 
-# FIXME: compile host tools using host-generic-pc target?
-b_log "Building hostutils"
-make -C "phoenix-rtos-hostutils" -f Makefile.old all
-cp "$PREFIX_BUILD_HOST/prog.stripped/phoenixd" "$PREFIX_BOOT"
-cp "$PREFIX_BUILD_HOST/prog.stripped/psdisk" "$PREFIX_BOOT"
-cp "$PREFIX_BUILD_HOST/prog.stripped/psu" "$PREFIX_BOOT"
-cp "$PREFIX_BUILD_HOST/prog.stripped/syspagen" "$PREFIX_BOOT"
+b_log "Installing hostutils"
+cp -a "$PREFIX_BUILD_HOST/prog.stripped/phoenixd" "$PREFIX_BOOT"
```

Comment:
not sure about copying different binaries per-target, maybe we could copy * to BOOT?

Reply:
Why not, too much tools shouldn't hurt anything and would make tool-adding process easier (e.g. I forgot you need to add it here too)

Reply:
Probably best would be to done it in hostutils Makefile but maybe it's not worthed (it makes hostutils install target different from other install targets). So ok if you copy *.

Reply:
- can't really do it in hostutils Makefile as I don't have the original TARGET value (so PHOENIX_BOOT path is invalid). I could try passing REAL_TARGET, etc. but IMHO it would be more confusing than beneficial
- went with explicit copying in `build.sh` (can't use `*` as there might be other programs in prog.stripped (eg. tests run on host)